test/test_smiley.jl

```diff
@@ -0,0 +1,37 @@
+include("smiley_examples.jl")
+
+using Base.Test
+using IntervalArithmetic, IntervalRootFinding
+using SmileyExample22, SmileyExample52, SmileyExample54, SmileyExample55
+
+function test_all_unique(xs)
+    for x in xs
+        @test x.status == :unique
+    end
+    return nothing
+end
+
+const tol = 1e-7
+const method = Newton # NOTE: Bisection method performs badly in all examples
```

Comment:
What do you mean by "performs badly"?

Reply:
I can't remember, either too slow or it missed roots. None of the examples worked and it seemed unhelpful to add them as @test_broken.